falkordb/asyncio/falkordb.py

```diff
@@ -55,7 +56,14 @@ def __init__(
             retry=None,
             connect_func=None,
             credential_provider=None,
-            protocol=2
+            protocol=2,
+            # FalkorDB Cluster Params
+            cluster_error_retry_attempts=3,
+            startup_nodes=None,
+            require_full_coverage=False,
+            reinitialize_steps=5,
+            read_from_replicas=False,
```

Comment:
Why False ?

Reply:
I was following the none async configuration